tests/test_tasks_downstream.py

```diff
@@ -128,6 +139,116 @@ def test_start(
             invoke("stop")
             stdout = invoke("start", "--debugpy")
             assert socket_is_open("127.0.0.1", int(supported_odoo_version) * 1000 + 899)
+            # Check if auto-reload is disabled
+            container_logs = docker_compose("logs", "odoo")
+            assert "dev=reload" not in container_logs
+    finally:
+        # Imagine the user is in the odoo subrepo for this command
+        with local.cwd(tmp_path / "odoo" / "custom" / "src" / "odoo"):
+            invoke("stop", "--purge")
+
+
+@pytest.mark.sequential
```

Comment:
Is this test really sequential? It doesn't seem so... :thinking: 

```suggestion
```

Reply:
Well, it requires bringing up the Odoo container, which depends on all the others that were causing problems with the `start` task tests.

Reply:
Yes but it opens no ports.